tests/spec/core/issues-notes-spec.js

```diff
@@ -160,6 +160,32 @@ describe("Core — Issues and Notes", function() {
     );
   });
 
+  it("removes closed issues from spec", async () => {
+    const githubConfig = {
+      github: "https://github.com/mock-company/mock-repository",
```

Comment:
this might conflict with the next property. Make sure you check that it's correctly overridden.

Reply:
Yeah sure, this was just a sort of demo

Reply:
@marcoscaceres, I think I do have to set `github` to something like `https://github.com/mock-company/mock-repository` otherwise it fires an error saying that github links should start with `https://github.com`. So I guess the conflict would persist :confused: